lib/alamo.js

```diff
@@ -403,18 +403,26 @@ async function create_service(pg_pool, service_name, plan, tags, space_name, app
   return await alamo_fetch('post', `${await get_region_api_by_space(pg_pool, space_name)}/v1/service/${service_name}/instance`, {plan, billingcode:tags})
 }
 
-async function create_osb_service(pg_pool, type, service_id, addon_service_id, plan_id, org_id, space_name, app_name) {
-  return await alamo_fetch('put', `${await get_region_api_by_space(pg_pool, space_name)}/v2/service_instances/${service_id}?accepts_incomplete=true`, {service_id:addon_service_id, plan_id, content:{platform:"akkeris", organization:{id:org_id}, space:{name:space_name}}, organization_guid:org_id})
+async function create_osb_service(pg_pool, type, service_id, plan_id, addon_id, org_id, space_name, app_name) {
```

Comment:
COBRA-2979 - A possible problem of not issuing the create/delete bindings during OSB creation may have resulted in the missing `REDIS_URL`.  This also standardizes the naming a bit.